BuffettCodeCommon/Registry/BuffettCodeRegistryAccessor.cs

```diff
@@ -0,0 +1,84 @@
+using BuffettCodeCommon.Exception;
+using Microsoft.Win32;
+using System.Linq;
+namespace BuffettCodeCommon.Registry
+{
+    public class BuffettCodeRegistryAccessor
+    {
+        private readonly string keyName;
+
+        private BuffettCodeRegistryAccessor(string keyName)
+        {
+            this.keyName = keyName;
+        }
+
+        private RegistryKey Open(bool writable) => Microsoft.Win32.Registry.CurrentUser.OpenSubKey(keyName, writable);
```

Comment:
📝 

> ローカル コンピューター (LocalMachine) よりも、現在のユーザー (CurrentUser) にデータを書き込む方が安全です。 作成しようとするキーが、以前に悪意のある可能性のある別のプロセスによって作成されたことがある場合、一般に "スクワッティング" と呼ばれる状況が発生します。 スクワッティングの発生を防ぐには、キーがまだ存在しない場合は Nothing を返す GetValue などのメソッドを使用します。

https://docs.microsoft.com/ja-jp/dotnet/visual-basic/developing-apps/programming/computer-resources/reading-from-and-writing-to-the-registry-using-the-microsoft-win32-namespace